packages/v4/src/content/design-guidelines/components/action-list/action-list.md

```diff
@@ -0,0 +1,30 @@
+---
+id: Action list
+section: components
+---
+An **action list** is a group of actions with set spacing.
+
+## Usage
+Action lists can be used in [toolbars](/components/toolbar), [modals](/components/modal), [forms](/components/form), [data lists](/components/data-list), [wizards](/components/wizard) and more. Using an action list allows you to know what spacing to use for a group of actions, depending on your use case. Although in most cases there should be 16px of spacing between buttons, there are cases where you may want more separation, such as with cancel buttons in wizards which should be placed 48px away from the rest of the actions. Using an action list ensures that these spacing guidelines are maintained and followed. 
```

Comment:
```suggestion
Use an action list to determine which spacing guidelines to use for a group of actions in [toolbars](/components/toolbar), [modals](/components/modal), [forms](/components/form), [data lists](/components/data-list), [wizards](/components/wizard), and more. 

Action lists are especially useful for situations that call for more than standard spacing between elements. For example, buttons are normally separated by 16px, but cases like cancel buttons in wizards call for 48px of spacing from other actions instead. Using an action list upholds spacing guidelines across different use cases.
```

Suggested restructuring for the introduction that combines the first two sentences into one and fleshes out a bit more context for the button use case example. I also rewrote the last sentence of avoid saying "ensure," since we try to avoid absolutes in most RH/PF content!